cli/src/command/wallet.rs

```diff
@@ -238,14 +238,22 @@ pub async fn node_info_command(storage_path: &Path) -> Result<Wallet, Error> {
 pub async fn restore_command(storage_path: &Path, snapshot_path: &Path, backup_path: &Path) -> Result<Wallet, Error> {
     check_file_exists(backup_path).await?;
 
-    let password = get_password("Stronghold password", false)?;
-    let secret_manager = SecretManager::Stronghold(
-        StrongholdSecretManager::builder()
-            .password(password.clone())
-            .build(snapshot_path)?,
-    );
-    let wallet = Wallet::builder()
-        .with_secret_manager(secret_manager)
+    let mut builder = Wallet::builder();
+    if check_file_exists(snapshot_path).await.is_ok() {
+        println_log_info!(
+            "Detected a stronghold file at {}. Enter password to unlock:",
+            snapshot_path.to_str().unwrap()
+        );
```

Comment:
Is that "Enter password to unlock:" necessary? Also a bit weird to have that in the log, no?

Reply:
yea its kind of needed, otherwise theres the chance you have to unlock 2 strongholds, and its not clear which one is which.

Reply:
I do agree its weird in the logs :/

Reply:
Then don't log it ?